run_dir/static/js/lanes_ordered.js

```diff
@@ -0,0 +1,284 @@
+            /* Delete the children from the data tree */
+            this.dumpData(key);
+        },
+        dumpData(key) {
+            let key1 = this.key_array[0];
+            let key2 = this.key_array[1];
+            let key3 = this.key_array[2];
+
+            // A function to delete everything except 'value' from a dictionary
+            function dumpDataHelper(dictionary) {
+                for (let [my_key, value] of Object.entries(dictionary)) {
+                    if (my_key !== 'value') {
+                        delete dictionary[my_key]
+                    }
+                }
+            }
+            switch (this.key_level) {
+                case 1: {
+                    // Delete everything except 'value' from statistics_data
+                    dumpDataHelper(this.$root.statistics_data[key])
+                    break;
+                }
+                case 2: {
+                    dumpDataHelper(this.$root.statistics_data[key1][key])
+                    break;
+                }
+                case 3: {
+                    dumpDataHelper(this.$root.statistics_data[key1][key2][key])
+                    break;
+                }
+            }
+        },
+        fetchData(key) {
+            let args = this.key_array.slice(0, this.key_level - 1);
+            args.push(key);
+            this.$root.fetchStatistics(...args);
+            this.$root.in_focus = args;
+        },
+        focus_on(key) {
+            let args = this.key_array.slice(0, this.key_level);
+            args.push(key);
+            this.$root.in_focus = args;
+        },
+        has_leaves(category) {
+            return (Object.keys(this.local_data[category]).length > 1) && (this.key_level < 5)
+        }
+    },
+    template:
+        /*html*/`
+        <template v-for="category in this.local_data_keys" :key="category">
+            <div :class="has_leaves(category) ? 'ml-4 mb-4' : 'ml-4'">
+                <h3 class="mb-2">
+                    <v-template v-if="key_level == 4">
+                        <a :href="'project/' + category">{{category}}</a> {{this.local_data[category]['value']}}
+                    </v-template>
+                    <v-template v-else>
+                        <v-template v-if="has_leaves(category)">
+                            <!-- If the category is expanded, show a collapse button -->
+                            <a href="#" @click="collapse(category)">
+                                <i class="fas fa-xs fa-minus mr-2"></i>
+                            </a>
+                            <!-- Clicking the category will only change focus of graph-->
+                            <a href="#" @click="focus_on(category)">
+                                {{category}}
+                            </a>
+                        </v-template>
+                        <v-template v-else>
+                            <a href="#" @click="fetchData(category)">
```

Comment:
Does this mean there's an ajax call each time a plus is clicked?

Reply:
Yes